modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -60,6 +66,8 @@
     ;; Spatial types -- see http://docs.oracle.com/cd/B28359_01/server.111/b28286/sql_elements001.htm#i107588
     [#"^SDO_"       :type/*]
     [#"STRUCT"      :type/*]
+    ;; TODO this regex seems to suggest there is a group?
+    [#"TIMESTAMP(\(\d\))? WITH TIME ZONE" :type/DateTimeWithTZ]
```

Comment:
many drivers have an optional precision for the `timestamp` type.

It could be `timestamp with time zone` or `timestamp(6) with time zone`